admin_manual/maintenance/code-signing.rst

```diff
@@ -0,0 +1,177 @@
+============
+Code Signing
+============
+
+.. sectionauthor:: Lukas Reschke <lukas@owncloud.com>
+
+ownCloud supports code signing for the core releases as well as dedicated applications. Code signing gives our users an
```

Comment:
Just cut the word »our users« here cause it’s too generic. »Code signing gives an additional layer of security …«